examples/nunjucks/assets/css/main.css

```diff
@@ -0,0 +1,4 @@
+/* stylelint-disable scss/at-rule-no-unknown */
```

Comment:
can't we define these "unknown" at rules in stylelint config or somewhere else, so there would not be a need for the ignore?

Reply:
Unlike EsLint, Stylelint doesn’t seem to accept an `overrides` section in its config, allowing to change rules for specific files, or at least I didn’t find something in that direction in the docs. Now I’m thinking it might accept sub-config files per folder, but it doesn’t really matter if we move that to another repo.

Reply:
Not overrides per se, but you should be able to define at-rules that should not throw: https://github.com/kristerkari/stylelint-scss/blob/HEAD/src/rules/at-rule-no-unknown/README.md#ignoreatrules-regex-string